### PR TITLE
Add tx2 on Jetpack 4.2 (Draft - DO NOT MERGE)

### PR DESCRIPTION
**Pull request by acostach**
- Add tx2 on Jetpack 4.2
- Update BalenaOS
- Update jetson nano

### CODE REVIEW

File: layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra186-flash-dry_32.1.0.bb [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+    mkdir -p ${DEPLOY_DIR_IMAGE}/bootfiles
+    local f
+    PATH="${STAGING_BINDIR_NATIVE}/tegra186-flash:${PATH}"
+    rm -rf "${WORKDIR}/tegraflash"
+    mkdir -p "${WORKDIR}/tegraflash"
+    oldwd=`pwd`
+    cd "${WORKDIR}/tegraflash"
+    ln -s "${STAGING_DATADIR}/tegraflash/${MACHINE}.cfg" .
+    ln -s "${IMAGE_TEGRAFLASH_KERNEL}" ./${LNXFILE}
```
Comment (acostach) [inline layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra186-flash-dry_32.1.0.bb]: final u-boot image written to kernel & kernel_b is generated by mkbootimg. we need a dependency for do_compile() for that task too.

File: layers/meta-balena-jetson/recipes-core/initrdscripts/jetson-tx2/resindataexpander [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```
Comment (floion) [inline layers/meta-balena-jetson/recipes-core/initrdscripts/jetson-tx2/resindataexpander]: Can we also have this initramfs module PRed in meta-balena too?

**Comment by floion** [comment]
Some file conflict apaprently in this Pr now since something was merged in master recently

**Comment by acostach** [comment]
Not needed anymore, many changes have been merged already, others will be merged when fixes are provided by l4t.

**Pull request closed without merging**